src/state/ducks/Quotes.ts

```diff
@@ -117,21 +107,26 @@ class Store {
       case Store.ACTIONS.SAVE_ATTESTERS_QUOTE: {
         const {
           quoteId,
-          claimerAddress,
+          owner,
           quote,
         } = (action as ISaveAttestersAction).payload
 
         return state.setIn(['quotes', quoteId], {
           quoteId,
-          claimerAddress,
+          owner,
           quote,
         } as Entry)
       }
       case Store.ACTIONS.SAVE_AGREED_QUOTE: {
-        const { quoteId, quote } = (action as ISaveAgreedQuoteAction).payload
+        const {
+          quoteId,
+          quote,
+          owner,
+        } = (action as ISaveAgreedQuoteAction).payload
 
         return state.setIn(['quotes', quoteId], {
           quoteId,
+          owner,
           quote,
         } as Entry)
       }
```

Comment:
can you tell me how these two actions are different? they seem identical to me on a superficial glance